### ISSUE TITLE
[Bug] QR Dosen't Show up when pressing F5

### ISSUE THREAD

**Issue by JeanTheFungle** [edqx/MouthwashPublic#1]
hey , i test mod but i found an issue is when you press F5 the qr dosen't show up i hope to fix this issue 

Contact me  in discord if you wante play together : jeanamongus 👍

**Comment by edqx** [comment]
I am not sure what to use the QR code for right now, but if you have any ideas then I'll take them.

**Comment by JeanTheFungle** [comment]
> I am not sure what to use the QR code for right now, but if you have any ideas then I'll take them

Yes i have many idea for the project i cant tell you here but i can tell you in discord

**Comment by edqx** [comment]
I meant ideas for the QR code specifically.
